ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -9,41 +10,54 @@
       destination_port: "{{ item.port }}"
       jump: ACCEPT
     with_items:
```

Comment:
looks like ` with_items`  is deprecated in favor of ` loop`  https://docs.ansible.com/ansible/2.5/user_guide/playbooks_loops.html